mappings/net/minecraft/world/gen/feature/CountFeatureConfig.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2984 net/minecraft/world/gen/feature/SeaPickleFeatureConfig
+CLASS net/minecraft/class_2984 net/minecraft/world/gen/feature/CountFeatureConfig
```

Comment:
Maybe just `CountConfig` since it implements both `FeatureConfig` and `DecoratorConfig`?